### PR TITLE
PLA-2941: accordions auto-collapse is fixed on resources tab

### PR DESCRIPTION
**Pull request by ariyDez**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -257,7 +257,7 @@
 
     <div id="resource-row-tmpl">
         <div class="resource-row {{if isOld}}old-data-mask{{/if}}">
-            <div class="view-row {{:engineStatusClass}}">
+            <div class="view-row {{:engineStatusClass}}" resource="{{:_id}}">
```
Comment (kimslava93) [inline index.html]: Do we really need `=` sign here?

File: js/deploy.js [hunk 2]
```diff
@@ -41,10 +43,20 @@ window.Deploy = (function () {
             appendLogs(resources[i].outputs, html.find('.logs .outputs .data-cont'));
         }
         initializeRowsActions();
+        resourcesFlag = utils.getResourceStatus();
+        for(var flag in resourcesFlag){
+            $("div[resource='"+flag+"']").addClass('opened');
```
Comment (kimslava93) [inline js/deploy.js]: `var t =  $("div[resource='"+flag+"']");
t.addClass('opened');
t.next('.expandable-row').removeClass('hidden-row');`

File: js/utils.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+var flagResources = {};
```
Comment (enigma91) [inline js/utils.js]: I m not sure that this is the right place for this data. Flags are used only in deploy part, why do not store data in deploy.js

File: index.html (hunk 1 above)
Comment (ariyDez) [inline index.html]: Yeap

File: js/deploy.js [hunk 4]
```diff
@@ -345,6 +351,14 @@ window.Deploy = (function () {
...
     }
 
+    function setResourceStatus(flags) {
+        flagResources = flags;
+    }
+
+    function getResourceStatus() {
+        return flagResources;
+    }
+
```
Comment (enigma91) [inline js/deploy.js]: these two function are not used anywhere

**Pull request merged**